sql/2020/08_Accessibility/title_used_with_alt.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# How often the title attribute is used with an alt attribute, and how often they are the same values
+SELECT
+  client,
+  SUM(total_alt) AS total_alts,
```

Comment:
Sometimes you do `COUNTIF(total > 0)`. Why not here?

Reply:
Because we're looking at the sheer number of alts overall, and not the individual sites themselves. So I have no need to filter out the sites which do not use alts.